### DISCUSSION TITLE
Video stream (like Twitch) on mobile devices?

### DISCUSSION THREAD

**Discussion by baeumer** [mozilla/hubs#5167] [General Questions & Discussion] [answered]
I added a video element with a Twitch URL (like https://www.twitch.tv/channelname).
It works on desktop / laptop browser but not on three different mobile devices I've tested (all three iOS, tested Safari as well as Firefox).

Should it work on mobile devices or is it not possible (and so only a desktop feature)?

Thanks for your help.

**Comment by pattersonbl2** [comment, marked as answer]
I can create an issue around this topic to look into it.  I have opened the issue here 
https://github.com/mozilla/hubs/issues/5184
